core/domain/exp_jobs_one_off.py

```diff
@@ -328,3 +328,32 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+class FallbackOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
Add extra newline above this one. In general there should be exactly two newlines above top-level class/function definitions.